ros2service/ros2service/verb/call.py

```diff
@@ -53,11 +52,29 @@ def main(self, *, args):
             raise RuntimeError('rate must be greater than zero')
         period = 1. / args.rate if args.rate else None
 
-        return requester(
-            args.service_type, args.service_name, args.values, period)
+        return requester(args.service_name, args.values, period)
+
+
+def requester(service_name, values, period):
+    rclpy.init()
 
+    node = rclpy.create_node(NODE_NAME_PREFIX + '_requester_%s' % (str(os.getpid())))
+    names_and_types = get_service_names_and_types(
+        node=node
+    )
+    for n, t in names_and_types:
+        if n == service_name:
+            if len(t) > 1:
```

Comment:
Does it make any sense to allow the user to pass the service type as an option?  Would that allow us to resolve this ambiguity in the case that the service has multiple types?

Reply:
Aye, I was very surprised to find a list being returned by `get_service_names_and_types()`. `ros2 topic echo` doesn't handle it so I'm curious if it is a use case I'm not aware of or just some relic of a previous architectural design.

Reply:
If we can't answer the question, we could a) re-insert the argument optionally to disambiguate, just in case, but I don't like adding complexity without understanding why or b) add a comment to advise if someone actually runs into the use case we are not aware of

Reply:
I discussed this with @stonier , and we came to the agreement that the current code is fine.  If we do end up in the situation where someone runs into the `RuntimeError` below, we can always lift the restriction then and add an option.